PSQL_README.md

```diff
@@ -0,0 +1,53 @@
+# How to Install PSQL
+## Debian / Ubuntu
+
+open the terminal and type in
+```bash
+sudo apt-get update 
+sudo apt-get install postgresql postgresql-contrib
+```
+
+## windows
+Download the installer from 
+    https://www.enterprisedb.com/downloads/postgres-postgresql-downloads
+and follow the installation process, if asked fill `5432` in port configuration
+
+# How to Configure the Database
+## Debian / Ubuntu
+Create `systersdb` database, where `systersdb` might be any suitable name.
+
+```bash
+sudo -i -u postgres
+createuser systersdb --pwprompt
```

Comment:
Just saying, systerdb is not a very intuitive name for a user :D but its fine too though.